node/_tools/suites/common/dns.js

```diff
@@ -5,10 +5,10 @@
 // Taken from Node 16.13.0
 // This file is automatically generated by "node/_tools/setup.ts". Do not modify this file manually
 
-'use strict';
+"use strict";
```

Comment:
I have no idea why this formatting wasn't being applied, all I did was run setup.ts and all this changes were applied